server/app/util/fbs/NetEncoding/Column.py

```diff
@@ -4,6 +4,7 @@
 
 import flatbuffers
```

Comment:
Everything in this file is machine generated and should not be reformatted or edited in any way.   With flake8's config, we ignore this directory.  Need to match that.

Reply:
Missed this. I've made sure the exclusion goes is used in the auto-formatting.